### PR TITLE
SVBONY CCD firmware and SDK version information logs added

### PR DESCRIPTION
**Pull request by jctk**
Added code to SVBONYCCD::Connect() to output the version of SVBONY Camera's firmware and SDK as information log.
This information is used when users inquire about issues.

### CODE REVIEW

File: indi-svbony/svbony_ccd.cpp [hunk 1]
```diff
@@ -219,6 +219,19 @@ bool SVBONYCCD::Connect()
...
     }
 
+    // Get Camera Firmware Version
+    status = SVBGetCameraFirmwareVersion(cameraID, cameraFirmwareVersion);
+    if (status == SVB_SUCCESS)
+    {
+        LOGF_INFO("Camera Firmware Version:%s", cameraFirmwareVersion);
+    }
+    else {
+        LOG_ERROR("Error, getting Camera Firmware Version failed.\n");
```
Comment (knro) [inline indi-svbony/svbony_ccd.cpp:229]: No need for \n

File: indi-svbony/svbony_ccd.h [hunk 2]
```diff
@@ -109,6 +109,11 @@ class SVBONYCCD : public INDI::CCD
         // pixel size
         float pixelSize;
 
+        // Camera Firmware version number
+        char cameraFirmwareVersion[65];
+        // SVBONY Camera SDK version number
+        const char* SDKVersion;
```
Comment (knro) [inline indi-svbony/svbony_ccd.h:115]: If you're only using this to report version information once in Connect, is there a reason to keep it as class variable? or is this used elsewhere?

File: indi-svbony/svbony_ccd.cpp (hunk 1 above)
Comment (jctk) [reply indi-svbony/svbony_ccd.cpp]: I committed it with newline code removed form LOG_ERROR.

File: indi-svbony/svbony_ccd.h (hunk 2 above)
Comment (jctk) [reply indi-svbony/svbony_ccd.h:115]: If EKOS is used, this information is output to the log and need not be retained.
If non-EKOS clients use it without logging, I will retain this information for reference when debugging.
We also plan to display this information in the INDI control panel in the future, so we decided to keep it.

**Pull request merged**